### PR TITLE
style: update language dropdown for dark mode

### PR DESCRIPTION
**Pull request by secondl1ght**
#### Type of change (Remove other not matching type)

- Style update

#### Describe the changes you have made in this PR -
Updated the new language dropdown selector element to work with dark mode.

#### Screenshots of the changes (If any) -
image

#### How Has This Been Tested?
Ran code locally and checked for dynamic style updates on dark/light theme switch in browser.

#### Checklist:

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Review by escapedcat — APPROVED** [review]

**Comment by dylancom** [comment]
Nice! Could you also remove the "h-14" so it matches height of our buttons?

File: src/app/components/form/Select/index.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ type Props = React.SelectHTMLAttributes<HTMLSelectElement> & {
 function Select({ children, value, name, onChange }: Props) {
   return (
     <select
-      className="w-full h-14 border-gray-300 rounded-md shadow-sm text-gray-700 bg-white"
+      className="w-full h-14 border-gray-300 rounded-md shadow-sm text-gray-700 bg-white dark:bg-gray-800 dark:text-gray-200 dark:border-gray-500"
```
Comment (dylancom) [inline src/app/components/form/Select/index.tsx:8]:
```suggestion
      className="w-full border-gray-300 rounded-md shadow-sm text-gray-700 bg-white dark:bg-gray-800 dark:text-gray-200 dark:border-gray-500"
```

Comment (dylancom) [reply src/app/components/form/Select/index.tsx:8]: Nice! Could you also remove the "h-14" so it matches height of our buttons?

Comment (secondl1ght) [reply src/app/components/form/Select/index.tsx:8]: Yep, I also thought the button looked a bit big.

**Pull request merged**